ImageGallery/src/org/sleuthkit/autopsy/imagegallery/datamodel/grouping/GroupManager.java

```diff
@@ -646,13 +646,22 @@ synchronized public void handleFileUpdate(Collection<Long> updatedFileIDs) {
             // reset the hash cache
             controller.getHashSetManager().invalidateHashSetsCacheForFile(fileId);
 
+            // first of all, update the current path group, regardless of what grouping is in view
+            try {
+                DrawableFile file = getDrawableDB().getFileFromID(fileId);
+                String pathVal = file.getDrawablePath();
+                GroupKey<?> pathGroupKey = new GroupKey(DrawableAttribute.PATH,pathVal, file.getDataSource());
```

Comment:
this is causing a unchecked call and raw type warning

Reply:
@rcordovano: The following PR addresses this warning, please merge it.

https://github.com/sleuthkit/autopsy/pull/4714